components/authorization/cluster/kcp-5ttoh8uv3y7i-admin.yaml

```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
```

Comment:
the file name could be more generic, eg. kcp-spi-namespace-admin.yaml, since you will also include there rolebinding for unstable kcp namespace.